features/zoek-met-geslachtsnaam-geboortedatum.feature

```diff
@@ -2,522 +2,416 @@
+    | naam                | waarde                              |
+    | type                | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam       | Aedel                               |
+    | geslachtsaanduiding | <geslachtsaanduiding>               |
+    | geboortedatum       | 1983-05-26                          |
+    | fields              | burgerservicenummer                 |
+    Dan bevat de response alleen personen met de volgende gegevens 
+    | burgerservicenummer |
+    | 999995084           |
+
+    Voorbeelden:
+    | geslachtsaanduiding |
+    | v                   |
+    | V                   |
+
+  @gba
+  Abstract Scenario: Zoek met geboorteplaats
+    Als personen wordt gezocht met de volgende parameters
+    | naam           | waarde                              |
+    | type           | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam  | Maassen                             |
+    | geboorteplaats | <geboorteplaats>                    |
+    | geboortedatum  | 1983-05-26                          |
+    | fields         | burgerservicenummer                 |
+    Dan bevat de response alleen personen met de volgende gegevens 
+    | burgerservicenummer |
+    | 999995082           |
+
+    Voorbeelden:
+    | geboorteplaats |
+    | Wassenaar      |
+    | wassenaar      |
+    | WASSENAAR      |
+
+  @gba
+  Scenario: Zoek met inclusiefOverledenPersonen
+    Als personen wordt gezocht met de volgende parameters
+    | naam                       | waarde                              |
+    | type                       | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam              | Jansen                              |
+    | inclusiefOverledenPersonen | true                                |
+    | geboortedatum              | 1983-05-26                          |
+    | fields                     | burgerservicenummer                 |
+    Dan bevat de response alleen personen met de volgende gegevens 
+    | burgerservicenummer |
+    | 999995081           |
+    | 999995085           |
+
+  @fout-case
+  Abstract Scenario: <titel>
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | <geslachtsnaam>                     |
+    | geboortedatum | <geboortedatum>                     |
+    | fields        | burgerservicenummer                 |
+    Dan bevat de response de volgende gegevens
+    | naam     | waarde                                                                                                                       |
+    | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest                  |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.                                                                    |
```

Comment:
volgens de Als stap worden wel de verplichte parameters geslachtsnaam en geboortedatum gestuurd maar met een lege waarde. Dan is er geen parametercombinatie fout, maar een pattern fout op deze parameters.

Wanneer de parameters niet gestuurd worden, verwacht ik een required fout, zoals dat in zoek-met-bsn.feature wordt afgehandeld.

Reply:
aangepast